src/components/search/Preview.tsx

```diff
@@ -4,14 +4,24 @@ import styled from 'styled-components';
 import { ICRecommand, ICVisible } from '../../asset/icon';
 import ImgHomePreview from '../../asset/image/previewImg.png';
 import { ContentPreviewProps, PreviewData } from '../../types/common';
+import Hover from '../common/Hover';
 
 const Preview = (props: ContentPreviewProps) => {
   const { contentPreviewData, isHomePage } = props;
   const { projectId, profileImg, name, recommandCount, visibleCount } = contentPreviewData;
 
+  const [isHover, setIsHover] = useState(false);
+  const handleHover = () => {
+    setIsHover(true);
+  };
+  const handleHoverOut = () => {
+    setIsHover(false);
+  };
```

Comment:
생각해보니까 이거 하나로 합칠수 있지안나
```suggestion ts
const handleHover = () => {
  setIsHover((prev)=>!prev);
}
```

Reply:
호오오오 이거 함수를 onmouseover랑 onmouseout으로 각각 생각했어서 따로 만들었었눈데 이렇게 함 고쳐볼게!!!

Reply:
히히 이거 내가 리뷰 이케 달았돈고지롱!